pkg/infrastructure/backend/dockerimpl/backend.go

```diff
@@ -180,9 +181,17 @@ func (b *Backend) containerLabels(app *domain.Application) map[string]string {
 		appLabel:            "true",
 		appIDLabel:          app.ID,
 		appRestartedAtLabel: app.UpdatedAt.Format(time.RFC3339Nano),
+		"sablier.enable":    lo.Ternary(b.useSablierMiddleware(app), "true", "false"),
+		"sablier.group":     sablierGroupName(app.ID),
 	})
 }
 
+func (b *Backend) useSablierMiddleware(app *domain.Application) bool {
```

Comment:
k8s impl の方を見た限り、関数名が違いそうだったのでこれだけ k8simpl と揃えたほうがいいのかなぁくらいです
```suggestion
func (b *Backend) useSablier(app *domain.Application) bool {
```